dependencies.sh

```diff
@@ -40,6 +231,11 @@ else
                 if [ $? == 0 ]
                 then
                        pip3 install -r requirements.txt
+                       if [ $? == 0 ]; then 
```

Comment:
remove apt install python3-pip. Python packages should be installed using pip3 from miniconda3 location. 
do the same for yum (centos) below too.